src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
```

Comment:
always set the font styles to the `html` tag. In `*` can be only `box-sizing`. Anything else is forbidden